pkg/gatherers/conditional/common_test.go

```diff
@@ -28,13 +29,8 @@ func Test_getAlertPodName(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			got, err := getAlertPodName(tt.labels)
-			if (err != nil) != tt.wantErr {
-				t.Errorf("getAlertPodName() error = %v, wantErr %v", err, tt.wantErr)
-				return
-			}
-			if got != tt.want {
-				t.Errorf("getAlertPodName() got = %v, want %v", got, tt.want)
-			}
+			assert.Equal(t, tt.wantErr, err != nil)
```

Comment:
There's `assert.NoError` or something like that.

Reply:
Well, this is true but then... I will have to use an if statement to cover `wantErr: true` and `wantErr: false`. Using `assert.Equal` one-line solution.

```go
if tt.wantErr {
    assert.Equal(t, tt.wantErr, true)
} else {
    !assert.NoError(t, tt.wantErr)
}
```
vs
```go
assert.Equal(t, tt.wantErr, err != nil)
```